ImprovedAnvil/scripts/areas/AR0410.baf

```diff
@@ -157,7 +157,9 @@ END
 IF
   InMyArea(Player1)
   Global("Iacells","AR0410",1)
-  XPGT(Player1,1999999)
+  OR(2)
+    XPGT(Player1,1999999)
+    Global("Iahardmode","Global",1)

```

Comment:
It's GLOBAL, uppercase

Reply:
well that was annoying

Reply:
You should at least do a compile test when you make changes, compilers are there for a reason - to catch these things :-)